App3/App3/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите двузначное число");
+        int number = int.Parse(Console.ReadLine());
+        int digits = number / 10;
+        int units = number % (digits * 10);
+        int sum = digits + units;
+        int mult = digits * units;
+
+        Console.WriteLine($"{digits}, {units}, {sum}, {mult}");
```

Comment:
С точки зрения пользователя непонятный вывод результатов на консоль
```suggestion
        Console.WriteLine($"Число десятков: {digits}, число единиц: {units}, сумма цифр: {sum}, произведение цифр: {mult}");
```